src/main/java/tools/MaplePacketCreator.java

```diff
@@ -70,9 +70,9 @@
 import java.net.InetAddress;
 import java.sql.ResultSet;
 import java.sql.SQLException;
-import java.util.List;
```

Comment:
Why did you remove java.util.List? Now we get this error instead:
`java: reference to List is ambiguous
  both interface java.util.List in java.util and class java.awt.List in java.awt match`